module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,18 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <array>
+
+bool isVowel(const char letter) {
+    constexpr int vowelsArraySize = 12;
+    constexpr std::array<char, vowelsArraySize> vowels = {'A', 'E', 'I', 'O', 'U', 'Y',
+                                                          'a', 'e', 'i', 'o', 'u', 'y'};
+
+    return std::find(vowels.begin(), vowels.end(), letter) != vowels.end();
+}
+
+void removeVowels(std::vector<std::string>& strings) {
```

Comment:
strings is const

Reply:
No it isn't

Reply:
@anraf1001 Correct, my dimwittedness, sorry.

Reply:
No problem